modules/cluster-logging-deploy-subscription.adoc

```diff
@@ -154,7 +146,7 @@ spec:
   name: "elasticsearch-operator"
 ----
 <1> You must specify the `openshift-operators-redhat` namespace.
-<2> Specify the `.status.channels[].name` value from the previous step.
+<2> Specify channel version `4.2`.
```

Comment:
so we're going to have to keep updating this doc w/ every release?  can we somehow make this more generic so we don't have that problem?

Reply:
Is there a plan to get the Elasticserch Operator to install using the web console? If users could install via the web console all of these steps become unnecessary.

Reply:
i thought there was some plan to have cluster logging installation automatically pull in elastic via a dependency process.

Reply:
afaik its possible to do everything from the console though not necessarily through the OLM bits.